### PR TITLE
Adds content and style to homepage

### PR DESCRIPTION
**Pull request by larathordar**
#### Description
Changes proposed in this pull request:

* Styling and content added to homepage

#### What I have learned working on this feature: 
* I learned to style a hompage

#### Screenshots:
screen shot 2018-04-12 at 19 47 29

### CODE REVIEW

File: src/pages/home/home.html [hunk 1]
```diff
@@ -1,14 +1,47 @@
 <ion-header>
     <ion-navbar>
       <ion-title>
+          {{myTitle}}
```
Comment (davidboll) [inline src/pages/home/home.html]: Why did you add myTitle ?

**Review by holgertidemand — APPROVED** [review]
I'm in love

**Review by holgertidemand — APPROVED** [review]

**Comment by holgertidemand** [comment]
Looks even better!

File: src/pages/home/home.html (hunk 1 above)
Comment (larathordar) [reply src/pages/home/home.html]: I don't know, will remove it

**Review by davidboll — APPROVED** [review]
Well done 💯

**Review by kmthorsnes — APPROVED** [review]
Looks goodie good!
